fabulaws-config.yml

```diff
@@ -2,6 +2,8 @@
   # "staging": staging
```

Comment:
should we update this line to make it clear that it's staging.presidentialopenquestions?